api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -33,11 +34,13 @@
 	private String name;
 	
 	private String description;
-	
+
 	private Set<Integer> memberIds;
-	
+
+	private Collection<CohortMembership> members;
+
 	public Cohort() {
-		memberIds = new TreeSet<Integer>();
+        members = new TreeSet<CohortMembership>();
```

Comment:
The formatting here does not look correct. Did you run mvn clean install before committing?